packages/tlstore/src/lib/Store.ts

```diff
@@ -461,6 +461,45 @@ export class Store<R extends UnknownRecord = UnknownRecord, Props = unknown> {
 		})
 	}
 
+	/**
+	 * Get a serialized snapshot of the store and its schema.
+	 *
+	 * ```ts
+	 * const snapshot = store.getSnapshot()
+	 * store.loadSnapshot(snapshot)
+	 * ```
+	 *
+	 * @public
+	 */
+	getSnapshot() {
+		return {
+			store: this.serialize(),
+			schema: this.schema.serialize(),
```

Comment:
Let's change this to filter out the non-document records by default. After my changes to the instance state management nobody should be creating snapshots that include the instance/presence state.

I have the code for this in my local branch and can add it here if you like.